src/clean_air/visualise/render_plot.py

```diff
@@ -4,15 +4,64 @@
 
 import hvplot.xarray  # noqa
 
+from iris.cube import Cube, CubeList
+import iris.plot
+import matplotlib as mpl
+from matplotlib import pyplot as plt
+
 
 class Plot:
-    """
-    Class to process data and create a lovely map from it.
-    """
     def __init__(self, dataframe):
-        self.df = dataframe.load()
+        # Note: This should always be a CubeList, even if it only contains
+        # one Cube.  This is just for simplification of the code.
+        self.dataframe = dataframe
 
     def render_timeseries(self):
-        # NOTE: Must have a a deployed Bokeh Server app or a deployed Panel
-        # app to be able to view these plots.
-        self.df.hvplot.line(datashade=True)
+        # First make a figure, then add all datasets as subplots:
+        fig = plt.figure(figsize=(11, 5))
+
+        # We want 3 plots per row unless there are less than 3 plots to
+        # render, then we want the figsize to match the number of plots:
+        n_plots = len(self.dataframe)
+        n_rows = int(n_plots-1/3) + 1
```

Comment:
I believe this needs bracketing like `int((n_plots-1) / 3) + 1`?

Reply:
Is this line equivalent to 
```suggestion
        n_rows = math.ceil(n_plots / 3)
```
or have I misunderstood your intention?

Reply:
You're right, this syntax is incorrect.  It should be this: int((n_plots - 1) /3) + 1

The intention is to produce a row number which is always 0 for up to 3 plots, always 1 for 6 plots, etc, so that we have three plots per line.